jac-uk/digital-platform
# Add extra personal fields to reports

## What's included?
Add the following fields to the personal details of reports (Character issues, Eligibility issues, Agency report, Handover report):
- Middle name(s) (optional)
- Suffix (optional)
- Previous known name(s) (optional)
- Professional name (optional)

## Who should test?
✅ Developers

## How to test?
Use postman to call the following endpoints to check if the above fields are exported:
- `exportApplicationCharacterIssues`
- `exportApplicationEligibilityIssues`
- `generateAgencyReport`
- `generateHandoverReport`

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
